index.js

```diff
@@ -138,6 +138,12 @@ export const startEvaluate = async ({
     })
   })
 
+  // Update top measurement stations every 12 hours
+  const pgClient = await createPgClient()
+  setInterval(async () => {
+    await pgClient.query('REFRESH MATERIALIZED VIEW top_measurement_participants_yesterday_mv')
+  }, 1000 * 60 * 60 * 12)
```

Comment:
> what do you think about the location of the refresh? It could be in bin/, or index.js as-is now, or some other non-timer trigger, like on each round-starts.

I think it's okay to have this code inside `startEvaluate()` function. It already installs other event listeners.

Just make sure `npm test` exits cleanly.